Lib/test/test_argparse/test_translations.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import re
+import subprocess
+import sys
+import unittest
+
+from pathlib import Path
+
+
+i18n_tools = Path(__file__).parents[3] / 'Tools' / 'i18n'
+pygettext = i18n_tools / 'pygettext.py'
+snapshot_path = Path(__file__).parents[0] / 'data' / 'msgids.txt'
+
+msgid_pattern = re.compile(r'msgid(.*?)(?:msgid_plural|msgctxt|msgstr)', re.DOTALL)
+msgid_string_pattern = re.compile(r'"((?:\\"|[^"])*)"')
```

Comment:
This does not work in the case like: ``` "abc\\"def" ```.
```suggestion
msgid_string_pattern = re.compile(r'"((?:\\.|[^"])*)"')
```

Reply:
On other hand, this should never happen in the PO files. So it is fine as is.